fission/function.go

```
@@ -29,6 +32,37 @@ import (
 	"github.com/fission/fission"
 )
 
+func _fnFetchCode(filePath string) []byte {
+	var code []byte
+	var err error
+
+	isURL, _ := regexp.MatchString("\\Ahttp(s|)://", filePath)
```

Don't ignore errors, please